docs/version0.27.md

```diff
@@ -0,0 +1,31 @@
+<!--
+* Copyright (c) 2017, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.27.0
+
+The following new features and notable changes since v 0.26.0 are included in this release:
+
+- [New `-XX:[+|-]AdaptiveGCThreading` option added](xxadaptivegcthreading.md)
```

Comment:
These topics normally include a short section about the change that you link to from the bullet. (See the topics for other versions as an example) I think I suggested some words that you might you. 

Other than this I am good with all your updates.

We're still finalising 0.26 before branching, so will hold this PR until we've done that.

Reply:
I've changed up the suggestion https://github.com/eclipse/openj9-docs/pull/765#issuecomment-815650471, how does this sound:
>To optimize performance, you can now enable adaptive threading to automatically tune the number of active parallel garbage collection (GC) threads. The gc thread count will be dynamically adjusted from collection cycle to cycle to account for changes in the workload and consider the available resources. When parallel workloads decrease, less threads can be used, reducing the overhead and freeing up resources for other processing activities.
>
>Use the [`-xgcmaxthreads`](xgcmaxthreads.md) option with the [`-XX:+AdaptiveGCThreading`](xxadaptivegcthreading.md) option to specify a thread count limit.

Reply:
I haven't commit this change yet. If this is good, then I'll go ahead and make the change

Reply:
@RSalman - couple of minor points. Please change gc -> GC and also remove the future tense, which we don't use in the docs. i.e. *When enabled, the GC thread count is dynamically adjusted......* 

Otherwise looks good and we should be able to merge and set 0.27.0 on its tracks today. Thanks for your efforts!